src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -697,10 +698,12 @@ export default {
       }
     },
 
-    async downloadDocument (document: any) {
+    async downloadDocument (document: any, index: number) {
       this.loadingDocument = true
+      this.downloadingDocIndex = index
       await this.downloadOneDocument(document)
```

Comment:
Should we also reset the index and loadingDocument indicator within the download error block as well?

Reply:
Those are getting reset after await this.downloadOneDocument(document) and there is not much processing in the error block other than setting the error dialog flag to true. Do we need that?

Reply:
I think the only reason would be if someone were to close the dialog to try again it would be the loading indicator instead of the file name.

Reply:
Ahh I get it now. Nvm it resets as it just passes through that code.